tests/test_utils.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime
+
+import pytest
+
+from devlog.utils.blog import post_from_markdown
+from devlog.utils.pagination import get_page
+
+
+def test_pagination_invalid_param(client, app):
+    val = 'dummy'
+    with app.test_request_context(f'/?p={val}'):
+        page = get_page()
+        assert page != val
+        assert page == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/jarek.zgoda/devlog/pullRequest?prid=5774733)